src/guides/v2.4/graphql/mutations/company-team-mutations.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Delete this file. We don't need it any more, now that the mutations are in separate topics.